api/v1beta1/kogitoinfra_types.go

```diff
@@ -103,13 +103,14 @@ func (k *KogitoInfraStatus) GetRuntimeProperties() api.RuntimePropertiesMap {
 	return runtimeProperties
 }
 
-// SetRuntimeProperties ...
-func (k *KogitoInfraStatus) SetRuntimeProperties(runtimeProperties api.RuntimePropertiesMap) {
-	newRuntimeProperties := make(map[api.RuntimeType]*RuntimeProperties, len(runtimeProperties))
-	for key, value := range runtimeProperties {
-		newRuntimeProperties[key] = value.(*RuntimeProperties)
+// AddRuntimeProperties ...
+func (k *KogitoInfraStatus) AddRuntimeProperties(runtimeType api.RuntimeType, runtimeProperties api.RuntimePropertiesInterface) {
```

Comment:
Nitpick, would be clearer to name the function `AddRuntimeProperty` as we pass just one property, WDYT?

Reply:
We actually pass here several properties in the struct, comment can be ignored.